client/packages/programs/src/JsonForms/common/components/Array/Notes.tsx

```diff
@@ -144,7 +144,11 @@ const NotesComponent = (props: ArrayControlCustomProps) => {
       return false;
 
     // Must not be older than `maxAge` days
-    if (DateUtils.ageInDays(created) >= 1) return false;
+    if (
+      DateUtils.ageInDays(created) >= options['editRestrictions']['maxAge'] ??
+      Infinity
+    )
+      return false;
```

Comment:
What about using the zod typing and update line 65 to

```
  const options = NotesOptions.parse(props.uischema.options);
```

then the options object is correctly typed, and ts forces you to make this change:

```suggestion
    if (
      DateUtils.ageInDays(created) >=
      (options['editRestrictions']['maxAge'] ?? Infinity)
    )
      return false;
```

Reply:
Okay, thanks 116702a30